src/main/resources/templates/macros/page/navigation.vm

```diff
@@ -7,9 +7,9 @@
         #if ($run_with_jenkins)
           <li role="presentation"><a href="../">Jenkins</a></li>
           #if ($build_previous_number)
-            <li role="presentation"><a href="../../$build_previous_number/cucumber-html-reports/$report_file">Previous results</a></li>
+            <li role="presentation"><a href="../../$build_previous_number/cucumber-html-reports$directory_suffix/$report_file">Previous results</a></li>

```

Comment:
looks good - did you check if old configuration with new plugin works fine? can you move back and forward between older and newer reports?

Reply:
When using the new version, if the suffix is not set or is set to an empty string, then navigation to previous reports work whether generated with the old or the new version. If a suffix is set, then there will not be "previous" report - so there won't be a link.  Previous/next is based on the same suffix.